<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-20485NC4" ignored="0" name="RestAssuredTest" tests="3" failures="1" timestamp="22 Apr 2020 07:22:11 GMT" time="30.414" errors="0">
  <testcase name="runScenario" time="16.778" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="6.235" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="5.476" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [300] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [300] but found [200]
at stepDefinition.ApiTestingStepDef.i_verify_the_status_code_as(ApiTestingStepDef.java:91)
at âœ½.I verify the status code as "300"(file:src/test/resources/features/APITesting_GetOrder.feature:7)
... Removed 5 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- RestAssuredTest -->
